### PR TITLE
log-backup: removed log-backup related configs

### PR DESCRIPTION
**Pull request by YuJuncen**
Signed-off-by: Yu Juncen <yujuncen@pingcap.com>

### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb/issues/29501

Problem Summary:
Log backup advancer provides some config values, which are obscure and maybe not suitable for proving to the user directly.

### What is changed and how it works?
This PR removed the config values for `advancer` in log backup.
We haven't make sure what arguments are needed to be exposed to end-users, for now, all configurable values would be the constant of default values.

When we did need to change some of them, we can use `br log advancer` and provide them by the CLI arguments.

### Check List

Tests 

- [ ] No code

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/streamhelper/config/advancer_conf.go [hunk 1]
```diff
@@ -39,20 +41,18 @@ type Config struct {
 }
 
 func DefineFlagsForCheckpointAdvancerConfig(f *pflag.FlagSet) {
-	f.Duration(flagBackoffTime, 5*time.Second, "The gap between two retries.")
-	f.Duration(flagMaxBackoffTime, 20*time.Minute, "After how long we should advance the checkpoint.")
-	f.Duration(flagTickInterval, 12*time.Second, "From how log we trigger the tick (advancing the checkpoint).")
-	f.Bool(flagAdvancingByCache, true, "Whether enable the optimization -- use a cached heap to advancing the global checkpoint.")
-	f.Int(flagFullScanDiffTick, 4, "The backoff of full scan.")
+	f.Duration(flagBackoffTime, DefaultBackOffTime, "The gap between two retries.")
+	f.Duration(flagTickInterval, DefaultTickInterval, "From how log we trigger the tick (advancing the checkpoint).")
```
Comment (3pointer) [inline br/pkg/streamhelper/config/advancer_conf.go:45]:
```suggestion
	f.Duration(flagTickInterval, DefaultTickInterval, "From how long we trigger the tick (advancing the checkpoint).")
```

**Review by 3pointer — COMMENTED** [review]
rest LGTM

**Review by joccau — APPROVED** [review]
LGTM

**Review by 3pointer — APPROVED** [review]
LGTM

**Pull request merged**